core/modules/widgets/button.js

```diff
@@ -271,6 +271,10 @@ ButtonWidget.prototype.refresh = function(changedTiddlers) {
 			sourcePrefix: "data-",
 			destPrefix: "data-"
 		});
+		this.assignAttributes(this.domNodes[0],{
+			sourcePrefix: "aria-",
+			destPrefix: "aria-"
+		});
```

Comment:
We should consider refactoring assignAttributes() to make it possible to pass in multiple prefixes as well as any other specific attributes that need to be assigned to a DOM node. Calling the function twice is sub-optimal unelegant and also sub-optimal in terms of performance. 

#9101 explores changes that touch upon this and one of the next steps likely involves a complete rewrite of assignAttributes for a cleaner and more intuitive function signature